### PR TITLE
Changing resistance equation

### PR DESCRIPTION
**Pull request by sfk-steelsong**
Hoped to get this in before bot tier went live, but life happens.  Better late than never!

As we discussed, I've verified everything but the battle image part, but I don't see why it _wouldn't_ work.

### CODE REVIEW

File: src/commands/commandList/battle/WeaponInterface.js [hunk 1]
```diff
@@ -517,15 +517,26 @@ module.exports = class WeaponInterface{
...
+			resRate = animal.animal.prr;
+		}
+		else {
+			res = animal.stats.mr;
+			resRate = animal.animal.mrr;
+		}
 		res = res[0]+res[1];
-		res = (res/(100+res))*.8;
+		res = (res/(200+res))*.7;
+		res = res + .3 * resRate / 15;
```
Comment (dpskane) [inline src/commands/commandList/battle/WeaponInterface.js:533]: multiplying by 0.3 and then dividing by 15 is the same as simply multiplying by 0.02

Comment (ChristopherBThai) [reply src/commands/commandList/battle/WeaponInterface.js:533]: The equation is fine. It's mostly for code legibility. However, can we add some parenthesis here? I'm getting paranoid of PEMDAS looking at this.

**Review by ChristopherBThai — CHANGES_REQUESTED** [review]

File: src/commands/commandList/battle/WeaponInterface.js (hunk 1 above)
Comment (sfk-steelsong) [reply src/commands/commandList/battle/WeaponInterface.js:533]: I like making you suffer, like you like making everyone suffer.

But, yeah, I can change it.

**Comment by FortitudeinGold** [comment]
Is it not possible that by lowering resistances like this you would inevitably buff gems, and thus cause a cascade of new balance issues when we haven't seen the current ones resolved? If you make 1 and 2 res pets lower you're encouraging higher res pets, certainly...but currently we've only 20 stat points to work with (assuming you're using cps, as you can get more creative this way) and by raising your resistance you lower your other stats.
For perspective, a healer currently needs points in wp, mag, and hp. To create a strong healer with res that compares to the existing equation, you'd need 3 or 4 res on *each*. That means even if you go low as possible, 3 3 res and 2 hp, you've still used 5 of your 14 stats, leaving you to balance 9 points across wp and mag. That leaves you with low wp, low mag, low hp, all without reaching the resistance values you were managing preupdate. As someone who's run gem aoe, vstaff comps, and dagger comps, meeting thousands of teams that could potentially kill them and constantly comparing what happened in fights, I can say with some certainty that aoe could well be king after this. The standard fish spider estaff sword comp would likely be in play, going first and damage rng would be huge, and lifesteal sword and vstaff would be their biggest counters. However, this does not sound far off from what we are currently working with: extremely strong comps that overtake the meta and prevent other comps from being viable, let alone shining. We are fixing dagger to make the meta less destructive to creative teams like sceptre orb or other passive comps like absolve, discharge, or even something like hgen sstaff. Dagger brutalises everything that doesn't protect perfectly or kill it quickly. By fixing dagger, only to replace it with this rework, we're not repairing the meta, merely shifting it. Were we to consider other options for reworking resistance, taking our time to watch the meta and focus on what needs nerfs or buffs (like tiffy), the game would much benefit. But I certainly don't think shaking up meta for shaking's sake is the right way to go about it, nor is it sane to throw it all off just to make koalalike pets viable. I also feel obligated to point out that many, no, most if not all players have been levelling pets, (often paid for themselves) over the course of literal years. Were you to take a misstep, throwing off their hardwon viability, you would likely find yourself with many displeased customers....

**Pull request closed without merging**